test/helpers/api-integration/v3/object-generators.js

```diff
@@ -33,6 +33,14 @@ export async function generateUser (update = {}) {
   return apiUser;
 }
 
+export async function updateUser (user, update = {}) {
```

Comment:
after generateUser is called you should be able to just do generatedUser.update since it's already wrapped in ApiIUser